exasol_advanced_analytics_framework/udf_communication/peer_communicator/peer_communicator_config.py

```diff
@@ -12,3 +12,4 @@ class PeerCommunicatorConfig:
     forward_register_peer_config: ForwardRegisterPeerConfig = ForwardRegisterPeerConfig()
     poll_timeout_in_ms: int = 200
     send_socket_linger_time_in_ms: int = 100
+    close_timeout_in_ms: int = 100000
```

Comment:
I guess this is an empiric value, and we could decrease later, if to big?

Reply:
It is more or less arbitrary. It is in the range of the abort timeout, which is also somewhat arbitrary. The idea is, waiting 1–2 minutes is not an issue.